pd-configuration-file.md

```diff
@@ -71,23 +71,6 @@ This document only describes parameters that are not included in command-line pa
 + Default value: `"pd-cluster"`
 + If multiple clusters that have nodes with same configurations are deployed successively, you must specify different tokens to isolate different cluster nodes.
 
-### `lease`
```

Comment:
why is this useless?

Reply:
It is barely changed by users directly, so I think we can hide it.

Reply:
@disksing @rleungx Any update?

Reply:
Ok, I am going to leave it for now.